src/dotnet/APIView/APIViewWeb/Client/src/reviews.ts

```diff
@@ -0,0 +1,26 @@
+﻿$(() => {
+  const searchBox = $("#searchBox");
+
+  // make the search box the initial focused element so users can just start typing once page loads
+  searchBox.focus();
+
+  const context = $(".review-name");
+  searchBox.keyup(function () {
+
+    // highlight matching text using mark.js framework
+    const searchText = (searchBox.val() as string).toUpperCase();
+    (context as any).unmark();
+    if (searchText) {
+      (context as any).mark(searchText, {
+        done: function () {
+          $(this).not(":has(mark)").hide();
```

Comment:
This is supposed to accomplish `    // hide rows that do not match`

Reply:
No, this is just for the highlighting.

Reply:
Fixed